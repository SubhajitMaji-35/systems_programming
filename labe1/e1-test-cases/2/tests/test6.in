1 23 456 7
89 5 12 300
9 100 4 77
2 8 300 50
