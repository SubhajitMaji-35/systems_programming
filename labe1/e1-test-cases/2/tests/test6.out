1 89 9 2 
23 5 100 8 
456 12 4 300 
7 300 77 50 
